src/pages/auth/login.tsx

```diff
@@ -29,12 +30,25 @@ const LoginPage = () => {
 
     try {
       const response = await UserService.login({ email, password });
-      if (response.message === 'Incorrect email or password') {
+      emailValidation(email, setEmailInvalid);
+      // create e toast message;
+      passwordValidation(password, setPasswordInvalid);
+      // create a toast message that say Password must have min 5 and max 20 characters with 1 letter, 1 number and 1 special character;
+      if (response.message) {
+        setEmailInvalid(true);
+        setPasswordInvalid(true);
+      } else if (response.user.email === email) {
+        setEmailInvalid(false);
+        setPasswordInvalid(false);
+        if (response.user.isEmailVerified === false) {
+          // create a toast message saying "Verify ur email account"
+        }
+      } else {
         setEmailInvalid(true);
         setPasswordInvalid(true);
       }
     } catch (error) {
-      console.log(error);
+      console.error(error);
```

Comment:
We should remove this console.error and handle the catch. We could have a toast call with a generic error message like: 'Oops. Something went wrong, please try again."